code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -13,11 +13,15 @@
 	var/active_offset = 0
 	/// What, if any, submap we render onto
 	var/map = ""
+	/// Due to a Byond bug where secondary maps have improperly offset render targets, we will need to override this to "LEFT,TOP" for clients >515.1614
+	var/relay_loc = "CENTER"
```

Comment:
It's better to document what this DOES rather then why it's here, since why it's here might change given time
If it doesn't and none finds another use for this we can just rip it out when we deprecate the older client versions

Reply:
i think i better explained